scripts/envDFL.sh

```diff
@@ -0,0 +1,243 @@
+    export DYNAFLOW_LAUNCHER_INSTALL_DIR=$DYNAFLOW_LAUNCHER_HOME/installLinux
+    export_var_env DYNAFLOW_LAUNCHER_SHARED_LIB=OFF # same default value as cmakelist
+    export_var_env DYNAFLOW_LAUNCHER_USE_DOXYGEN=ON # same default value as cmakelist
+    export_var_env DYNAFLOW_LAUNCHER_BUILD_TESTS=ON # same default value as cmakelist
+    export_var_env DYNAFLOW_LAUNCHER_CMAKE_GENERATOR="Unix Makefiles"
+    export_var_env DYNAFLOW_LAUNCHER_PROCESSORS_USED=1
+
+    # Run
+    export DYNAFLOW_LAUNCHER_INSTALL=$DYNAFLOW_LAUNCHER_INSTALL_DIR
+    export LD_LIBRARY_PATH=$DYNAFLOW_LAUNCHER_HOME/lib64:$LD_LIBRARY_PATH
+    export_var_env DYNAFLOW_LAUNCHER_LOG_LEVEL=INFO # INFO by default
+
+    # other
+    set_commit_hook
+}
+
+reset_environment_variables() {
+    for var in $(printenv | grep DYNAFLOW_LAUNCHER_ | cut -d '=' -f 1); do
+        unset $var
+    done
+
+    ld_library_path_remove $DYNAFLOW_LAUNCHER_HOME/lib64
+    ld_library_path_remove $DYNAFLOW_LAUNCHER_HOME/lib
+
+    unset DYNAWO_HOME
+    unset DYNAWO_INSTALL_DIR
+    unset DYNAWO_RESOURCES_DIR
+    unset DYNAWO_DDB_DIR
+    unset DYNAWO_IIDM_EXTENSION
+    unset DYNAWO_LIBIIDM_EXTENSIONS
+    unset IIDM_XML_XSD_PATH
+}
+
+clean() {
+    rm -rf $DYNAFLOW_LAUNCHER_BUILD_DIR
+    rm -rf $DYNAFLOW_LAUNCHER_INSTALL_DIR
+}
+
+cmake_configure() {
+    cmake \
+        -S $DYNAFLOW_LAUNCHER_HOME \
+        -B $DYNAFLOW_LAUNCHER_BUILD_DIR \
+        -G $DYNAFLOW_LAUNCHER_CMAKE_GENERATOR \
+        -DCMAKE_BUILD_TYPE:STRING=$DYNAFLOW_LAUNCHER_BUILD_TYPE \
+        -DCMAKE_INSTALL_PREFIX:STRING=$DYNAFLOW_LAUNCHER_INSTALL_DIR \
+        -DDYNAWO_HOME:STRING=$DYNAWO_HOME \
+        -DBOOST_ROOT:STRING=$DYNAWO_HOME \
+        -DDYNAFLOW_LAUNCHER_LOCALE:STRING=$DYNAFLOW_LAUNCHER_LOCALE \
+        -DDYNAFLOW_LAUNCHER_SHARED_LIB:BOOL=$DYNAFLOW_LAUNCHER_SHARED_LIB \
+        -DDYNAFLOW_LAUNCHER_USE_DOXYGEN:BOOL=$DYNAFLOW_LAUNCHER_USE_DOXYGEN \
+        -DDYNAFLOW_LAUNCHER_BUILD_TESTS:BOOL=$DYNAFLOW_LAUNCHER_BUILD_TESTS
+}
+
+cmake_build() {
+    cmake --build $DYNAFLOW_LAUNCHER_BUILD_DIR --target install -j $DYNAFLOW_LAUNCHER_PROCESSORS_USED
+}
+
+cmake_tests() {
+    pushd $DYNAFLOW_LAUNCHER_BUILD_DIR
+    ctest -j $DYNAFLOW_LAUNCHER_PROCESSORS_USED --output-on-failure
+    popd
+}
+
+cmake_coverage() {
+    ctest \
+        -S cmake/CTestScript.cmake \
+        -DDYNAWO_HOME=$DYNAWO_HOME \
+        -DBOOST_ROOT=$DYNAWO_HOME \
+        -VV
+}
+
+clean_build_all() {
+    clean
+    build_user
+}
+
+build_user() {
+    cmake_configure
+    cmake_build
+}
+
+build_tests_coverage() {
+    rm -rf $DYNAFLOW_LAUNCHER_HOME/buildCoverage
+    cmake_coverage
+}
+
+launch() {
+    $DYNAFLOW_LAUNCHER_INSTALL_DIR/bin/DynaFlowLauncher \
+    --log-level $DYNAFLOW_LAUNCHER_LOG_LEVEL \
+    --network $1 \
+    --config $2
+}
+
+version() {
+    $DYNAFLOW_LAUNCHER_INSTALL_DIR/bin/DynaFlowLauncher --version
+}
+
+launch_dir() {
+    network=$1/network.xml
```

Comment:
I would have expected launch $network $cnfig and the file checks put into launch()

Reply:
launch_dir option allows to launch predefined files from a directory. it relies on launch option. No check is performed.

Reply:
I am refering to the tests that make sure the files exist

Reply:
These checks are done in DFL

Reply:
OK